data_steward/test/unit_test/vocabulary_test.py

```diff
@@ -8,9 +8,9 @@
 
 from test.unit_test import test_util
 from common import DELIMITER, LINE_TERMINATOR
-from resources import AOU_GENERAL_CONCEPT_CSV_PATH
+from resources import AOU_VOCAB_CONCEPT_CSV_PATH
```

Comment:
If you implement the possible suggestions, this test case will have to be updated.